janabimustafa/IDB
# made production build run on nginx instead of node

This addresses the trello ticket: https://trello.com/c/RYcMTwYp/18-update-the-react-version-to-run-on-production-version-instead-of-the-build-version

To test, run 
```
docker-compose -f docker-compose-travis.yml build
docker-compose -f docker-compose-travis.yml up -d
```
Going to your Docker-machine ip address (or localhost for non-Windows) should result in the site running on a production instance of Nginx instead of a development node server. 

This change should only apply when merging to master, and when the site deploys to rldb.me. This PR shouldn't change anything when it comes to dev.rldb.me (you will still be able to see error logs there).
